scripts/forestadmin-schema-update.js

```diff
@@ -7,4 +7,8 @@ if (!process.env.FOREST_ENV_SECRET || !process.env.FOREST_AUTH_SECRET) {
   process.exit();
 }
 
-forest.init();
+async function main() {
+  await forest.init();
+}
```

Comment:
[Top-level await](https://github.com/tc39/proposal-top-level-await) is at stage 3. Soon!

Reply:
Before writing that, I looked if babel could support it. They support the syntax, but not the feature.